src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -315,7 +322,14 @@ private void settings(final int buttonNo) {
         ObservableList<Node> buttons;
         buttons = rootNode(stage).getScene().getRoot()
                 .getChildrenUnmodifiable();
-        clickOn(buttons.get(buttonNo), MouseButton.PRIMARY);
+        if (buttonNo <= SETTINGS_SOUNDEFFECTS) {
+            clickOn(buttons.get(buttonNo), MouseButton.PRIMARY);
+        } else {
+            clickOn(buttons.get(buttonNo), MouseButton.PRIMARY);
+            press(MouseButton.PRIMARY);
+            moveTo(MOVE_X_SLIDER, MOVE_Y_SLIDER);
```

Comment:
Maybe it's safer to use `moveBy(double, double)` here. Using this, the E2E will also properly function on smaller screens, as `moveTo` moves to an absolute position on the screen. After that, you can also revert the `assertEquals(0, volume)` that failed earlier.

<testing, logic